examples/06_manipulating_images/plot_smooth_mean_image.py

```diff
@@ -24,13 +28,77 @@
 first_epi_file = data.func[0]
 
 # First compute the mean image, from the 4D series of image
-mean_func = image.mean_img(first_epi_file, copy_header=True)
+mean_func = mean_img(first_epi_file, copy_header=True)
 
 # %%
 # Then we smooth, with a varying amount of smoothing, from none to 20mm
 # by increments of 5mm
 for smoothing in range(0, 25, 5):
-    smoothed_img = image.smooth_img(mean_func, smoothing)
-    plotting.plot_epi(smoothed_img, title=f"Smoothing {int(smoothing)}mm")
+    smoothed_img = smooth_img(mean_func, smoothing)
+    plot_epi(
+        smoothed_img,
+        title=f"Smoothing {int(smoothing)}mm",
+        colorbar=True,
+        cmap="gray",
+        vmin=0,
+    )
+
+show()
+
+
+# %%
+from nilearn.datasets import (
+    load_fsaverage,
+    load_fsaverage_data,
+    load_sample_motor_activation_image,
+)
+from nilearn.plotting import plot_surf_stat_map
+from nilearn.surface import SurfaceImage
+from nilearn.surface import smooth_img as smooth_surf_img
+
+fsaverage_meshes = load_fsaverage()
+
+stat_img = load_sample_motor_activation_image()
+
+curvature = load_fsaverage_data(data_type="curvature")
+
+surface_image = SurfaceImage.from_volume(
+    mesh=fsaverage_meshes["pial"],
+    volume_img=stat_img,
+)
+
+hemi = "right"
+
+for iteration in range(0, 25, 5):
```

Comment:
We cannot specify smoothing kernels in number of iterations. We should specify them in terms of fwhm.